### DISCUSSION TITLE
How to use elements in angular?

### DISCUSSION THREAD

**Discussion by subodh-badavate** [muxinc/elements#669] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Which Mux Elements/Packages does this apply to? Select all that apply

mux-video

### Which browsers are you using?

Chrome

### Which operating systems are you using?

None

### Description

How can we use <mux-video> tag in angular projects.

### Steps to reproduce

1. add npm @mux/mux-video package in angular project
2. Add mux-video tag in html file.
3. mux-video element not found in project.

### Current Behavior

mux-video element not found in angular project.

### Expected Behavior

mux-video element should be found in angular project.

**Comment by gkatsev** [comment]
Hi, we have a codesandbox example using angular you can check out here: https://codesandbox.io/s/mux-player-angular-6qy8p7?file=/src/app/app.component.ts

That uses mux-player specifically, but you should be able to swap it out with `@mux/mux-video`.
